70 Add API annotations description

Benkoff opened the pull request:
connect to #70

Commit:
#70 initial swagger files added

billMeLater commented on docs/links/swagger.json:
```diff
@@ -0,0 +1,130 @@
+  "host": "evo.juja.com.ua",
+  "basePath": "/v1",
+  "schemes": [
+    "https"
```
so far we agreed on http, https is not supported

Benkoff replied:
as I can see there is https in fact: https://evo.juja.com.ua

Benkoff replied:
label removed, thanks. as well as the model names fixed. btw we've got Link model isn't it better to call it LinkRequestResponse? too many Links are bit confusing as for me

billMeLater commented on docs/links/swagger.json:
```diff
@@ -0,0 +1,130 @@
+          "type": "string",
+          "description": "the link been saved before"
+        },
+        "label": {
```
not aware about label. Dont remember we decided to have that

Commit:
Removed label, PostRequest renamed to fit the model as SaveLinkRequest, LinkRequestResponse corresponds to Link model

Commit:
#70 Refactored to exclude divergence with actual code

it's mostky similar but has some extra descriptions and examples. SaveLinkRequest API model corresponds to Link model in the code, could be renamed later.

Benkoff commented:
there are typos in the last commit. please read:
it's mostly similar but has some extra descriptions and examples. ~~SaveLinkRequest~~ LinkRequestResponse API model corresponds to Link model in the code, could be renamed later.

billMeLater reviewed (dismissed):
looks good

Commit:
#70 Add API annotations description, remove Swagger config files

Benkoff commented:
after our latest decision to use annotations instead of files configuration, those files are deleted and an API annotations description added.
a readable copy: https://github.com/Benkoff/juja-platform/blob/develop/docs/links/api.md

Commit:
#70 Add more samples, fix bugs and typos

Commit:
#70 Improve the final sentence readability

adaleorion reviewed (approved):
thanks
